_data/projects/OpinionatedGuides.yml

```
@@ -0,0 +1,11 @@
+name: Opinionated Guides
+desc: a one-stop guide to computers, linux, and electronics.
+site: https://github.com/opinionatedguide/opinionatedguide.github.io-src
+tags:
+- awesome
+- documentation
+- python
+- bash
+upforgrabs:
+  name: hacktoberfest
```

This needs to be an exact match of the label in use: 

```suggestion
    name: Hacktoberfest
```